src/Esfa.Vacancy.Api.Types/CreateApprenticeshipParameters.cs

```diff
@@ -16,5 +18,15 @@ public class CreateApprenticeshipParameters
         /// A description of the vacancy.
         /// </summary>
         public string LongDescription { get; set; }
+
+        /// <summary>
+        /// The closing date of the applicaiton.
```

Comment:
spelling?

I would document this as VacancyCloseDate. I'm just thinking about the rubbish names which we can keep out of our codebase maybe.

Reply:
IMHO, given that this date represents the last date on which applications can be submitted for the vacancy so using the word "application" is apt.